src/window.blp

```diff
@@ -449,17 +449,17 @@ Gtk.ApplicationWindow window {
             StackPage {
               name: "inline";
               child: Box {
-                ScrolledWindow {
```

Comment:
unfortunally AdwOverlay lost it when inside a `ScrolledWindow` but removing it lets the user mess up Workbench too easily.

TBD